### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by Lozu**
Solves:
	Assignment [Nov 26]: Registration - adding accounts to CredentialsManager + PR

### CODE REVIEW

File: app/src/main/java/com/example/mobileapps/LoginActivity.kt
Comment (luodongchen) [inline app/src/main/java/com/example/mobileapps/LoginActivity.kt:1]: it will continue the process if the user input empty email and pw

**Review by luodongchen — APPROVED** [review]

File: app/src/main/java/com/example/mobileapps/LoginActivity.kt
Comment (Lozu) [reply app/src/main/java/com/example/mobileapps/LoginActivity.kt:1]: Are you sure? I checked and with empty email/password I see small error indicator at the right side of the field.

**Comment by luodongchen** [comment]
No problem 👌

File: app/src/main/java/com/example/mobileapps/LoginActivity.kt
Comment (Lozu) [reply app/src/main/java/com/example/mobileapps/LoginActivity.kt:1]: Resolved

**Pull request merged**